rust-app/src/implementation.rs

```diff
@@ -545,7 +559,12 @@ fn handle_tx_param_1 (
     match tx_type {
         0 => {
             write!(hasher, ",\"payload\":{{\"exec\":{{\"data\":{{}},\"code\":\"").ok()?;
-            write!(hasher, "(coin.transfer \\\"k:{}\\\"", pkh_str).ok()?;
+            if namespace_str.is_empty() {
+                write!(hasher, "(coin.transfer").ok()?;
+            } else {
+                write!(hasher, "({}.{}.transfer", namespace_str, mod_name_str).ok()?;
+            }
```

Comment:
What about things like this?
```suggestion
            write!(hasher, "(").ok()?;
            if namespace_str.is_empty() {
                write!(hasher, "coin").ok()?;
            } else {
                write!(hasher, "({}.{}", namespace_str, mod_name_str).ok()?;
            }
            write!(hasher, ".transfer").ok()?;
```